XamlControlsGallery/ControlPages/TextBlockPage.xaml

```diff
@@ -13,6 +13,12 @@
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:local="using:AppUIBasics"
         xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="d">
+    <Page.Resources>
+        <Style TargetType="TextBlock" x:Key="CustomTextBlockStyle">
+            <Setter Property="Foreground" Value="{ThemeResource SystemAccentColor}"/>
```

Comment:
Accent color text looks like a hyperlink. Instead of color, what if this set a specific FontFamily instead? Something obvious and silly, like Comic Sans?

Reply:
That's a good point. Switched to fontfamily comic sans now instead.